packages/bodiless-psh/README.md

```diff
@@ -166,16 +166,17 @@ requests, so that every PR to your repository will be deployed to its own
 environment. Be aware of the following:
 - It is easy to reach your quota of development environments with this enabled, so use cautiously.
   One way to limit environment use is to create *draft* pull requests, as these are not deployed.
-- Edit environments for pull requests are currently only supported on GitHub repos -- and pushing 
-  changes from the edit environment is not supported on cross repository pull request(e.g. PR's
-  source branch from fork).
+- Edit environments for pull requests are currently only supported on GitHub repos. Content editor
+  can make page updates on these edit environments and commit/push the changes back to remote
+  GitHub repo. *note:* push feature is not available for cross repository pull requests
+  (e.g., PR's source branch is from forked repo).
```

Comment:
I think it's worth adding a note that for PR environments, it is the source branch of the PR which is checked out on p.sh.